.detekt.yml

```diff
@@ -3,8 +3,8 @@
 # https://github.com/arturbosch/detekt/blob/master/detekt-cli/src/main/resources/default-detekt-config.yml.
 
 complexity:
-  ComplexMethod:
-    threshold: 33
+  CyclomaticComplexMethod:
```

Comment:
Please explain why for undocumented declaration the single line approach makes more sense?

I don't think this is obvous, in particular in cases where a second annotation is added to a declaration or in case declaration pass constructor args where the call takes multiple lines.

Reply:
> Please explain why for undocumented declaration the single line approach makes more sense?

Undocumented field declarations in our code base are often not separated by blank lines, i.e. they come in the form

    @SerialName("npm")
    NPM,
    @SerialName("crate")
    CRATE,
    @SerialName("git")
    GIT,
    @SerialName("maven")
    MAVEN,
    ...

Instead of blowing up the code length by adding blank lines like in

    @SerialName("npm")
    NPM,

    @SerialName("crate")
    CRATE,

    @SerialName("git")
    GIT,

    @SerialName("maven")
    MAVEN,

    ...

I decided to follow @mnonnenmacher's preferred style (introduced with `PythonInspector`) to have the annotation and declaration compact in a single line:

    @SerialName("npm") NPM,
    @SerialName("crate") CRATE,
    @SerialName("git") GIT,
    @SerialName("maven") MAVEN,
    ...

> If decision for single line approach has been made already, please feel free to ignore this comment

I don't think a decision has been made yet, but IMO the single-line style should at least be allowed.